components/service/glean/docs/metrics_ping.md

```diff
@@ -0,0 +1,6 @@
+# The `metrics` ping
+The metrics ping contains all of the metrics defined in metrics.yaml that don't specify a ping or 
+where "default" is specified in their pings list.
+
+The `metrics` ping is automatically sent when the application is moved to the *background* and it 
```

Comment:
Please describe the full logic here, the fact that it's sent only once every 24 hours and how that works by checking the time when we get to background.